.github/workflows/build-and-push.yml

```
@@ -0,0 +1,44 @@
+name: Build and Push Docker Image
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - 'apps/**'
+      - 'docker/explorer/**'
+  pull_request:
+    paths:
+      - 'apps/**'
+      - 'docker/explorer/**'
+
+jobs:
+  build-and-push:
+    name: Build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: [linux/amd64, linux/arm64]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Log in to Amazon ECR
+        uses: aws-actions/amazon-ecr-login@v1
+        with:
+          registry-ids: "${{ secrets.AWS_ACCOUNT_ID }}"
+
+      - name: Build and push Docker image
+        run: |
+          docker buildx build \
+            --platform linux/${{ matrix.arch }} \
+            --tag $ECR_REPOSITORY:explorer-${{ matrix.docker_tag_suffix }}-${{ github.sha }} \
```

```suggestion
            --tag $ECR_REPOSITORY:dwallet-explorer-${{ matrix.docker_tag_suffix }}-${{ github.sha }} \
```
what is `docker_tag_suffix` ? I don't see it in the matrix above.

add it and put: "v1.16.2" for a linux/amd64
and "v1.16.2-arm" for linux/arm64